docs/gaia/gaiacli.md

```diff
@@ -87,15 +87,42 @@ Note that this is the Tendermint signing key, _not_ the operator key you will us
 We strongly recommend _NOT_ using the same passphrase for multiple keys. The Tendermint team and the Interchain Foundation will not be responsible for the loss of funds.
 :::
 
-#### Multisig public keys
+#### Generate multisig public keys
 
 You can generate and print a multisig public key by typing:
 
 ```bash
-gaiacli show --multisig-threshold K name1 name2 name3 [...]
+gaiacli keys add --multisig=name1,name2,name3[...] --multisig-threshold=K new_key_name
 ```
 
-`K` is the minimum weight, e.g. minimum number of private keys that must have signed the transactions that carry the generated public key.
+`K` is the minimum number of private keys that must have signed the
+transactions that carry the public key's address as signer.
+
+The `--multisig` flag must contain the name of private or public keys that will be combined into a
+public key that will be generated and stored as `new_key_name` in the local database. All names
+supplied through `--multisig` must already exist in the local database. The order in which the keys
+are supplied on the command line matters, i.e. the following commands generate two distinct
+multisig keys:
+
+```bash
+gaiacli keys add --multisig=foo,bar,baz --multisig-threshold=2 multisigaddress
+```
+
+```bash
+gaiacli keys add --multisig=baz,foo,bar --multisig-threshold=2 multisigaddress
+```
+
+Although keys and threshold parameter used in the above examples are the same, the commands
```

Comment:
Can we sort the names? This behaviour seems unusual, unless there's a specific reason we want it.

Reply:
... actually, can we sort by the public keys? that way different names on different machines will still work

Reply:
This sounds like a good idea. Ideally order doesn't matter here.

Reply:
yeah I second this. I can see simple mishaps happening due to this.

Reply:
Sorted - sorry for the pun.

Reply:
I've also added the flag `--nosort` so that a user can pass the keys in the exact order he wants and expect they don't get shuffled.